scheduler/src/cook/mesos/dru.clj

```diff
@@ -69,20 +69,19 @@
       scored-tasks)
     '()))
 
-(defn compute-gpu-task-scored-task-pair
+(defn compute-sorted-task-cumulative-gpu-score-pairs
   "Takes a sorted seq of task entities and the gpu divisor, returns a list of [task cumulative-gpus], preserving the same order of input tasks"
-  [task-ents gpu-divisor]
-  (let [task-resources (->> task-ents
-                            (map (comp #(select-keys % [:gpus]) util/job-ent->resources :job/_instance)))
-        task-cum-gpus (->> task-resources
-                           (accumulate-resources)
-                           (map (fn [{:keys [gpus]}]
-                                  (/ gpus gpu-divisor))))
-        scored-tasks (map (fn [task cum-gpus]
-                            [task cum-gpus])
-                          task-ents
-                          task-cum-gpus)]
-    scored-tasks))
+  [gpu-divisor task-ents]
+  (if (seq task-ents)
```

Comment:
I kept the behavior and structure same as `compute-task-scored-task-pairs` which has the same if branching.